lib/fae/helper_texter.rb

```diff
@@ -0,0 +1,20 @@
+class HelperTexter
```

Comment:
The way you implemented this will only make use of it when using Fae generators. I'm wondering if a better spot is in `Fae::FormHelper` where you could call a private method from `fae_input` that sets these defaults? What do you think @jasonfine?